sql/core/src/test/scala/org/apache/spark/sql/CollationStringExpressionsSuite.scala

```diff
@@ -17,135 +17,130 @@
 
 package org.apache.spark.sql
 
-import org.apache.spark.{SparkConf, SparkIllegalArgumentException}
-import org.apache.spark.sql.catalyst.expressions.{ExpressionEvalHelper, Literal, StringTrim, StringTrimLeft, StringTrimRight}
-import org.apache.spark.sql.catalyst.util.CollationFactory
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.catalyst.expressions._
 import org.apache.spark.sql.internal.SQLConf
 import org.apache.spark.sql.test.SharedSparkSession
-import org.apache.spark.sql.types.{ArrayType, BinaryType, BooleanType, DataType, IntegerType, StringType}
+import org.apache.spark.sql.types._
 
 // scalastyle:off nonascii
 class CollationStringExpressionsSuite
-  extends QueryTest
-  with SharedSparkSession
-  with ExpressionEvalHelper {
+    extends QueryTest
```

Comment:
Why did you add the indents. The style guide says about 2 spaces, see https://github.com/databricks/scala-style-guide?tab=readme-ov-file#spacing-and-indentation. Does scala style check complain about it?

Reply:
fixed